hydromt_wflow/wflow.py

```diff
@@ -622,21 +625,21 @@ def setup_lulcmaps(
         """
         self.logger.info(f"Preparing LULC parameter maps.")
         if lulc_mapping_fn is None:
-            fn_map = join(DATADIR, "lulc", f"{lulc_fn}_mapping.csv")
+            fn_map = f"{lulc_fn}_mapping"
         else:
             fn_map = lulc_mapping_fn
-        if not isfile(fn_map):
-            self.logger.error(f"LULC mapping file not found: {fn_map}")
-            return
+        if not isfile(fn_map) and fn_map not in self.data_catalog:
```

Comment:
To avoid conflict with user data (eg user defined corine_mapping source), maybe we should name the parameters data corine_mapping_default or something so here if fn_map is not found try for {lulc_fn}_mapping_default as well before returning an error?
Because the parameters_data catalog is read last  think it has priority on source names compared to the other data catalogs... When in core we solve issue [#329](https://github.com/Deltares/hydromt/issues/329) then we can be more precise.